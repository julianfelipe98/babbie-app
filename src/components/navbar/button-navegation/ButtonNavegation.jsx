import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import BottomNavigation from '@material-ui/core/BottomNavigation';
import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';

const useStyles = makeStyles({
  root: {
    width: "50%",
  },
});
export const BottonNavegaton = () => {
    const classes = useStyles();
    const [value, setValue] = React.useState(0);
  
    return (
      <BottomNavigation
        value={value}
        onChange={(event, newValue) => {
          setValue(newValue);
        }}
        showLabels
        className={classes.root}
      >
        <BottomNavigationAction label="Inicio"  />
        <BottomNavigationAction label="Producto" />
        <BottomNavigationAction label="Contacto"  />
      </BottomNavigation>
    );
  
}
