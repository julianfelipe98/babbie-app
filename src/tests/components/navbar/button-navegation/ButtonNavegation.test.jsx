import { shallow } from "enzyme";
import { ButtonNavegation } from "../../../../components/navbar/button-navegation/ButtonNavegation";
describe("ButtonNavegation component tests", () => {
  test("should render <ButtonNavegation/> in the correct way", () => {
    const wrapper = shallow(<ButtonNavegation />);
    expect(wrapper).toMatchSnapshot();
  });
});
