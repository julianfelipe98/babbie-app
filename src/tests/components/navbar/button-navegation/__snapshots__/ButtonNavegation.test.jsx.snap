// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonNavegation component tests should render <ButtonNavegation/> in the correct way 1`] = `
<WithStyles(ForwardRef(BottomNavigation))
  className="makeStyles-root-1"
  onChange={[Function]}
  showLabels={true}
  value={0}
>
  <WithStyles(ForwardRef(BottomNavigationAction))
    label="Inicio"
  />
  <WithStyles(ForwardRef(BottomNavigationAction))
    label="Producto"
  />
  <WithStyles(ForwardRef(BottomNavigationAction))
    label="Contacto"
  />
</WithStyles(ForwardRef(BottomNavigation))>
`;
